### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by ferhatelmas**
Was enjoying [missing link in postgres](https://brandur.org/postgres-default) and saw   

> Did I make a mistake? Please consider sending a pull request.

and here we are.

Thanks for good write-up.

### CODE REVIEW

File: content/fragments/patronage.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ hook: Thoughts on how to make the creation of high quality
   content on the internet sustainable.
 ---
 
-The world's changing, and moreso for independent creatives
+The world's changing, and more for independent creatives
```
Comment (brandur) [inline content/fragments/patronage.md]: Can you change this to "more so"?

File: content/fragments/turning-points.md [hunk 2]
```diff
@@ -20,4 +20,4 @@ successful relationship looks like, and in reality it may involve functions
 that don't have much romantic appeal like a heavy dose of daily familiarity
 compounded over an extended period of time. A relationship that's allowed to
 hobble its way to a slow demise is costly, but a false negative tripped by an
-overdeveloped internal cynic might be even moreso.
+overdeveloped internal cynic might be even more.
```
Comment (brandur) [inline content/fragments/turning-points.md]: And "more so" here too please.

**Comment by ferhatelmas** [comment]
@brandur updated per your request.

**Comment by brandur** [comment]
Thank you again!

**Pull request merged**